repository/cowsay/tests/cowsay-config-map/00-assert.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kudo-controller-manager-0
+  namespace: kudo-system
+status:
+  phase: Running
+
+# confirms that kudo is running
```

Comment:
Why is it necessary to check this here?

Reply:
I also thought that [this PR](https://github.com/kudobuilder/kudo/pull/1592) removed this assertion in favor of `kudo init ... --wait`

Reply:
it makes the test more portal IMO... yes we should have a kudo init --wait.  This fails fast if that wasn't done.. and I see these tests as something we may point users to... that may run them...

Reply:
I mean, we don't wait for KUDO _this way_ for other operators tests *today* (because we already `kudo init --wait`) so I'd rather stay consistent right now and remove this. The inconsistency is IMO worse than portability concerns.

Reply:
I agree with @zen-dog here - This is testing something that should be ensured globally not in each test. If we have this here, it will be copied to other tests and we end up with having this step duplicated everywhere. 

We had this in the kudo e2e tests as well, I was glad we could remove it there.